packages/relayer-lib/src/events/eureka.rs

```diff
@@ -17,14 +18,24 @@ use super::cosmos_sdk;
 /// Events emitted by IBC Eureka implementations that the relayer is interested in.
 #[derive(Debug, Clone, PartialEq, Eq)]
 #[allow(clippy::module_name_repetitions)]
+pub struct EurekaEventWithHeight {
+    /// The type of the event.
+    pub event: EurekaEvent,
+    /// The block number at which the event was emitted.
+    pub block_number: Option<u64>,
```

Comment:
Maybe in a separate PR, we can have this stop being an option, and emit an error instead. If the log doesn't have a height, I somehow feel like the event hasn't been mined.